tests/helpers.py

```diff
@@ -168,6 +170,7 @@ def generate_fake_data_query(rows=1000, null_percentage=0) -> str:
             CAST('abc' AS NTEXT) Col_Ntext,
             CAST('abc' AS NVARCHAR(100)) Col_Nvarchar,
             CAST('a' AS NCHAR) Col_Nchar,
+            CAST('55044B34-7F3A-48B5-9F70-18B68EB1C500' AS VARCHAR) Col_GUID,
```

Comment:
`CAST('55044B34-7F3A-48B5-9F70-18B68EB1C500' AS VARCHAR)` is casting to a string type so isn't testing correctly. I think you need to do `CAST('55044B34-7F3A-48B5-9F70-18B68EB1C500' AS UNIQUEIDENTIFIER)` instead.